test/soap.test.js

```diff
@@ -53,7 +53,7 @@ describe('loopback:soap tests', function() {
             content = readModelJsonSync('globals');
             expect(content).to.not.have.property('public');
             expect(content).to.have.property('properties');
-            expect(content.properties.Promise.type).to.eql('boolean');
+            expect(content.properties.EventEmitter.type).to.eql('boolean');
```

Comment:
@emonddr did the PR strongloop/loopback-soap#31 trigger this change in tests ? can you give some details ?

Reply:
@deepakrkris , sure. 

The historical test case was slightly wrong.

It was committing the same check twice:

```
content = readModelJsonSync('globals');
            expect(content).to.not.have.property('public');
            expect(content).to.have.property('properties');
            expect(content.properties.Promise.type).to.eql('boolean');
            expect(content.properties.Promise.type).to.eql('boolean');

```
It was checking that Promise.type is equal to boolean  `twice` .

The model it was checking was:

```
           <s:element name="GetQuote">
                <s:complexType>
                    <s:sequence>
                        <s:element minOccurs="0" maxOccurs="1" name="symbol" type="s:string"/>
                        <s:element name="globals" type="Globals"/>
                    </s:sequence>
                </s:complexType>
            </s:element>
            <s:complexType name="Globals">
                <s:sequence>
                    <s:element name="EventEmitter" type="s:boolean"/>
                    <s:element name="Promise" type="s:boolean"/>
                </s:sequence>
            </s:complexType>
```

it was forgetting to check

```
<s:element name="EventEmitter" type="s:boolean"/>
```

Reply:
@deepakrkris , did I address your question?

Reply:
So this change isn't actually needed, it's more of a cleanup?

Might be better to do that in a separate PR or at least as a separate commit within this PR.

Reply:
@rmg , I have placed the 'test case cleanup' in its own commit. :)